### PR TITLE
docs: Correct endpoint Url for creation of Ticket Discount Code in Docs.

### LINKED ISSUE #6260: Wrong Url in Docs for Ticket Discount Code
**Describe the bug**
Incorrect [Url](https://api.eventyay.com/#discount-codes-ticket-discount-code-collection-post) is defined in Docs for creating Ticket Discount Code.

**Additional context**
On it.

### PR DESCRIPTION
**Pull request by prateekj117**
Fixes #6260 

#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [X] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)

- [X] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by kushthedude** [comment]
Please change the prefix to `docs:`

File: docs/api/api_blueprint.apib [hunk 1]
```diff
@@ -17483,7 +17483,7 @@ Create a new Discount Code for event. (Only Admin)
     + sort (optional, string, `code`) - Sort the resources according to the given attribute in ascending order. Append '-' to sort in descending order.
     + filter (optional, string, `[]`) - Filter according to the flask-rest-jsonapi filtering system. Please refer: http://flask-rest-jsonapi.readthedocs.io/en/latest/filtering.html for more.
 
-### Create Ticket Discount Code [POST]
+### Create Ticket Discount Code [POST /v1/discount-codes{?page%5bsize%5d,page%5bnumber%5d,sort,filter}]
```
Comment (kushthedude) [inline docs/api/api_blueprint.apib]: This change would also be needed for `Create Access Code`

Comment (prateekj117) [reply docs/api/api_blueprint.apib]: @kushthedude Why?

Comment (kushthedude) [reply docs/api/api_blueprint.apib]: For the access code, Only `POST` is written without the exact URL .

Comment (prateekj117) [reply docs/api/api_blueprint.apib]: @kushthedude Read API Blueprint documentation. And before giving reviews, make sure you understand the issue and the solution implemented.

**Comment by prateekj117** [comment]
@iamareebjamal Please review.

**Pull request merged**